manim/scene/scene_file_writer.py

```diff
@@ -330,7 +331,30 @@ def add_sound(
 
         """
         file_path = get_full_sound_file_path(sound_file)
-        new_segment = AudioSegment.from_file(file_path)
+        # we assume files with .wav / .raw suffix are actually
+        # .wav and .raw files, respectively.
+        if file_path.suffix not in (".wav", ".raw"):
+            wav_file_path = NamedTemporaryFile(suffix=".wav")
+            with (
+                av.open(file_path) as input_container,
+                av.open(wav_file_path, "w", format="wav") as output_container,
```

Comment:
It seems on Windows `NamedTemporaryFile` cannot be opened twice (it's opened once on creation). One of the things [the docs](https://docs.python.org/3/library/tempfile.html#tempfile.NamedTemporaryFile) suggest is setting `delete=False`, or setting `delete_on_close=False` and unlinking manually.
Unfortunately, I don't have a Windows machien to test this on, so I can't give a better suggestion :)